Libplanet/Tx/Transaction.cs

```diff
@@ -75,7 +79,7 @@ public Transaction(ITxMetadata metadata, IAction systemAction, byte[] signature)
         public Transaction(ITxMetadata metadata, IEnumerable<T> customActions, byte[] signature)
         {
             _metadata = new TxMetadata(metadata);
-            CustomActions = customActions.ToImmutableList();
+            _customActions = customActions.Select(ca => ca.PlainValue).ToImmutableList();
```

Comment:
I'm not sure about this part. I made the `Transaction<T>` store all actions as `Bencodex.Types.IValue` type for state consistency. Other way to avoid serialization well is this, https://github.com/planetarium/libplanet/commit/f009ef723abee82f19dd91425c40b76dc3fe3e24